references/cli/addon.go

```diff
@@ -166,7 +166,11 @@ Enable addon for specific clusters, (local means control plane):
 				}
 				ioStream.Infof("enable addon by local dir: %s \n", addonOrDir)
 				// args[0] is a local path install with local dir, use base dir name as addonName
-				name = filepath.Base(addonOrDir)
+				abs, err := filepath.Abs(addonOrDir)
+				if err != nil {
+					return errors.Wrapf(err, "cannot open directory %s", addonOrDir)
```

Comment:
```suggestion
					return errors.Wrapf(err, "directory %s is invalid", addonOrDir)
```

How about changing it as above? We not trying to `open it`, so the log might be confusing.